src/test/java/wedlog/address/logic/commands/CommandTestUtil.java

```diff
@@ -126,6 +130,12 @@ public class CommandTestUtil {
         DESC_BOB = new EditPersonDescriptorBuilder().withName(VALID_NAME_BOB)
                 .withPhone(VALID_PHONE_BOB).withEmail(VALID_EMAIL_BOB).withAddress(VALID_ADDRESS_BOB)
                 .withTags(VALID_TAG_HUSBAND, VALID_TAG_FRIEND).build();
+        DESC_VAL = new EditVendorDescriptorBuilder().withName(VALID_NAME_VAL)
+                .withPhone(VALID_PHONE_AMY).withEmail(VALID_EMAIL_AMY).withAddress(VALID_ADDRESS_AMY)
+                .withTags(VALID_TAG_FRIEND).build();
+        DESC_BRYAN = new EditVendorDescriptorBuilder().withName(VALID_NAME_VICTOR)
```

Comment:
Why does BRYAN have the name VICTOR? Perhaps DESC_BRYAN should be renamed to DESC_VICTOR